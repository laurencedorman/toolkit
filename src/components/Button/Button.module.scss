/**
 * Copyright (c) Colibri SAS - ManoMano
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

@import '../../styles/import.scss';

.button {
  position: relative;
  display: inline-flex;
  justify-content: center;
  align-items: center;
  min-height: em(48);
  padding: em($smallSpace) em($space);
  text-align: center;
  border: em($border-width) $border-style transparent;
  border-radius: em($radius);
  background: transparent;
  font-size: em(16);
  line-height: 1.2;
  cursor: pointer;
  transition: background-color 0.15s ease, color 0.15s ease;
  touch-action: manipulation;

  &.default,
  &.primary {
    color: $white;
  }

  &.default {
    background-color: $balataGreen;
    &:hover {
      background-color: $articCitric;
    }
    &:active {
      background-color: $balataGreen;
    }
  }

  &.link {
    color: $balataGreen;
    background: none;
    border: none;
    padding: 0;
    min-height: 1em;
    height: 1em;
    transition: color 0.2s ease;

    &:hover {
      color: $balataGreenHover;
      background: none;
      text-decoration: underline;
    }
  }

  &.primary {
    background-color: $bellaNotte;
    &:hover {
      background-color: #2952b1;
    }
    &:active {
      background-color: $bellaNotte;
    }
  }

  &.secondary {
    color: $squidInk;
    background-color: $white;
    border-color: $greyDarker;

    &:hover {
      background-color: $greyDark;
    }
    &:active {
      background-color: $white;
      border-color: $greyDarker;
    }
  }

  &.ghost {
    color: $balataGreen;
    background-color: $white;
    border-color: $balataGreen;

    &:hover {
      background-color: $balataGreen;
      color: $white;
    }

    &:active {
      background-color: $white;
      border-color: $greyDarker;
      color: $balataGreen;
    }

    &:disabled {
      background-color: $greyLight;
      border-color: $greyMedDarker;
      color: $white;
    }
  }

  &.reset {
    background-color: transparent;
    border: none;
    color: $squidInk;

    &:hover {
      background-color: $greyDark;
    }
    &:active {
      background-color: $white;
    }
  }

  &:disabled,
  &:disabled:hover {
    background-color: $greyDarker !important;
    border-color: $greyDarker !important;
  }

  //SIZE SPACE
  &.big {
    min-height: em(56);
    padding: em($space);
  }

  &.small {
    min-height: em(40, 14);
    padding: calc(#{em($space)} / 4) em($space);
  }

  &.xsmall {
    min-height: em(32, 14);
    padding: em(4, 14) em($space);
  }

  &.small,
  &.xsmall {
    font-size: em(14);
    svg {
      width: em(12, 14);
    }
  }

  //ICON SPACE
  &.icon svg {
    margin-right: em($smallSpace);
  }

  &.reverse {
    flex-direction: row-reverse;
    svg {
      margin-right: 0;
      margin-left: em($smallSpace);
    }
  }

  &.onlyIcon svg {
    margin: 0;
  }
}
