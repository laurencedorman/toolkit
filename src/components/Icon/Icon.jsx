// @flow
import React from 'react';
import cn from 'classnames';

import iconAssets from '../../assets/setIcons';
import styles from './Icon.module.scss';

/**
 * @visibleName Icon
 */
const IconComponent = ({
  onClick, className, name, size, fill, dataQa,
}: propTypes) => {
  const classNames = cn(
    styles.container,
    className,
  );

  return (
    <div
      className={classNames}
      onClick={onClick}
      onKeyPress={onClick}
      tabIndex="-1"
      role="button"
      data-qa={dataQa}
    >
      {Object.prototype.hasOwnProperty.call(iconAssets, name)
      && (
        <svg
          className={styles.svg}
          width={`${String(size)}px`}
          height={`${String(size)}px`}
          viewBox={iconAssets[name].viewBox}
          aria-describedby={name}
          fill={fill}
          xmlns="http://www.w3.org/2000/svg"
          role="img"
        >
          <path
            d={iconAssets[name].d}
          />
        </svg>
      )}
    </div>
  );
};

type propTypes = {
  onClick?: () => void,
  className?: string,
  name: string,
  size?: number,
  fill?: string | () => void,
  dataQa?: string,
};

IconComponent.defaultProps = {
  onClick: null,
  className: '',
  size: 32,
  fill: '#0c193a',
  dataQa: '',
};

export default IconComponent;
