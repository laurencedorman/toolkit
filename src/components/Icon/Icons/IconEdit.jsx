import React from 'react';

const IconEdit = () => (
  <path d="M10.8361 23.8663L22.3743 12.233l-2.6036-2.6035L8.1689 21.1992l2.6672 2.6671zm-1.4084 1.42l-2.675-2.6749L5 24.3594v2.6324h2.7362l1.6915-1.7054zm14.355-14.4734l2.9744-2.999-2.5752-2.5836-2.995 2.9868 2.5958 2.5958zM8.5695 28.9919H3.6a.6.6 0 0 1-.6-.6v-4.8625L23.4778 3.1079c.3911-.39 1.0243-.3891 1.4143.002l3.9834 3.9963c.3883.3895.3891 1.0196.0018 1.4102L8.5695 28.9918z" />
);

export default IconEdit;
