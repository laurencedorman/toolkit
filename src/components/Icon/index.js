export { default } from './Icon';
export { default as Svg } from './Svg';
export { default as Img } from './Img';
