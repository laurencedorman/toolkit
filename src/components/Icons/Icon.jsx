// @flow
import React from 'react';
import styles from './Icon.module.scss';
import icons from '../../assets/setIcons';

const Icon = ({ name, size, fill }: propTypes) => (
  <svg
    className={styles.svg}
    width={`${size}px`}
    height={`${size}px`}
    viewBox={icons[name].viewBox}
    aria-labelledby={icons[name]}
    xmlns="http://www.w3.org/2000/svg"
    role="img">
    <path
      d={icons[name].d}
      fill={fill} />
  </svg>
);

type propTypes = {
  name: string,
  size?: number,
  fill?: string,
};

Icon.defaultProps = {
  size: 32,
  fill: '#0c193a',
};

export default Icon;
