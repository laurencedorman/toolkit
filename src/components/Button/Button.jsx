// @flow
import React from 'react';
import Icon from '../Icon';
import cn from 'classnames';
import styles from './Button.module.scss';

/**
 * @visibleName Button
 */
const Button = ({
  children, size, theme, disabled, type, icon, reverse, onClick, dataQa }: propTypes) => {
  const className = cn(
    styles.manoolkitButton,
    styles[size],
    styles[theme],
    {
      [styles.icon]: icon,
      [styles.reverse]: reverse,
      [styles.onlyIcon]: !children,
    }
  );

  return(
    <button
      disabled={disabled}
      type={type}
      onClick={onClick}
      className={className}
      data-qa={dataQa}
    >
      {icon &&
      <Icon name={icon} size="16" />}
      {children && children}
    </button>
  );
};

type propTypes = {
  children?: string | Node,
  theme?: 'default' | 'primary' | 'secondary',
  size?: 'big' | 'default' | 'small' | 'xsmall',
  type?: 'submit' | 'reset' | 'button' | 'menu',
  icon?: string,
  reverse?: boolean,
  disabled?: boolean,
  onClick?: () => void,
  dataQa?: string,
};

Button.defaultProps = {
  children: null,
  theme: 'default',
  size: 'default',
  type: 'button',
  icon: '',
  reverse: false,
  disabled: false,
  onClick: null,
  dataQa: '',
};

export default Button;
