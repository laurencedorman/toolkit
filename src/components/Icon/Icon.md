Icon:

- Import component icon with props <b>component</b>.
- You need to import Icon component and IconName.
- Icon was in SVG.
- Open a Merge Request if you need new icon.

  <br />

Import example:

```jsx static
import { Icon, IconAccount } from 'ToolkitPath';

<Icon component={IconAccount} />;
```

```js
import IconAbTest from './Icons/IconAbTest';
import IconAccount from './Icons/IconAccount';
import IconAlertCircle from './Icons/IconAlertCircle';
import IconAnalyse from './Icons/IconAnalyse';
import IconArrowFullDown from './Icons/IconArrowFullDown';
import IconArrowFullLeft from './Icons/IconArrowFullLeft';
import IconArrowFullRight from './Icons/IconArrowFullRight';
import IconArrowFullUp from './Icons/IconArrowFullUp';
import IconAward from './Icons/IconAward';
import IconBankTransfert from './Icons/IconBankTransfert';
import IconBarredEye from './Icons/IconBarredEye';
import IconBasket from './Icons/IconBasket';
import IconBathTub from './Icons/IconBathTub';
import IconBigDrill from './Icons/IconBigDrill';
import IconBox from './Icons/IconBox';
import IconBoxes from './Icons/IconBoxes';
import IconBrush from './Icons/IconBrush';
import IconBurgerMenu from './Icons/IconBurgerMenu';
import IconCalendar from './Icons/IconCalendar';
import IconCamera from './Icons/IconCamera';
import IconCheck from './Icons/IconCheck';
import IconChevronDown from './Icons/IconChevronDown';
import IconChevronLeft from './Icons/IconChevronLeft';
import IconChevronRight from './Icons/IconChevronRight';
import IconChevronUp from './Icons/IconChevronUp';
import IconCircleMinus from './Icons/IconCircleMinus';
import IconCirclePlus from './Icons/IconCirclePlus';
import IconClock from './Icons/IconClock';
import IconClose from './Icons/IconClose';
import IconCloseCircle from './Icons/IconCloseCircle';
import IconContent from './Icons/IconContent';
import IconCreditCard from './Icons/IconCreditCard';
import IconDeliveryForm from './Icons/IconDeliveryForm';
import IconDocument from './Icons/IconDocument';
import IconDotMenu from './Icons/IconDotMenu';
import IconDownload from './Icons/IconDownload';
import IconDragNdrop from './Icons/IconDragNdrop';
import IconDrill from './Icons/IconDrill';
import IconEdit from './Icons/IconEdit';
import IconExtra from './Icons/IconExtra';
import IconEye from './Icons/IconEye';
import IconFacebook from './Icons/IconFacebook';
import IconFaucetWater from './Icons/IconFaucetWater';
import IconFeatureFlag from './Icons/IconFeatureFlag';
import IconFilter from './Icons/IconFilter';
import IconFlash from './Icons/IconFlash';
import IconFreeDelivery from './Icons/IconFreeDelivery';
import IconHeart from './Icons/IconHeart';
import IconHeartFull from './Icons/IconHeartFull';
import IconHelpCircle from './Icons/IconHelpCircle';
import IconHistoricalSearch from './Icons/IconHistoricalSearch';
import IconHood from './Icons/IconHood';
import IconInformationCircle from './Icons/IconInformationCircle';
import IconInstagram from './Icons/IconInstagram';
import IconLabel from './Icons/IconLabel';
import IconLamp from './Icons/IconLamp';
import IconLaunch from './Icons/IconLaunch';
import IconLink from './Icons/IconLink';
import IconLock from './Icons/IconLock';
import IconLogOut from './Icons/IconLogOut';
import IconMail from './Icons/IconMail';
import IconMapPin from './Icons/IconMapPin';
import IconMarketing from './Icons/IconMarketing';
import IconMessage from './Icons/IconMessage';
import IconMessageCircle from './Icons/IconMessageCircle';
import IconPhone from './Icons/IconPhone';
import IconPiggyBank from './Icons/IconPiggyBank';
import IconPinterest from './Icons/IconPinterest';
import IconPlay from './Icons/IconPlay';
import IconPrinter from './Icons/IconPrinter';
import IconProduct from './Icons/IconProduct';
import IconPromo from './Icons/IconPromo';
import IconScrew from './Icons/IconScrew';
import IconSearch from './Icons/IconSearch';
import IconStarEmpty from './Icons/IconStarEmpty';
import IconStarFull from './Icons/IconStarFull';
import IconStop from './Icons/IconStop';
import IconTagPromo from './Icons/IconTagPromo';
import IconThumbDown from './Icons/IconThumbDown';
import IconTrash from './Icons/IconTrash';
import IconTrending from './Icons/IconTrending';
import IconTrucks from './Icons/IconTrucks';
import IconTwitter from './Icons/IconTwitter';
import IconVote from './Icons/IconVote';
import IconWall from './Icons/IconWall';
import IconWateringCan from './Icons/IconWateringCan';
import IconWheel from './Icons/IconWheel';
import IconYoutube from './Icons/IconYoutube';

import IconBelgium from './Icons/IconBelgium';
import IconFrance from './Icons/IconFrance';
import IconGermany from './Icons/IconGermany';
import IconItaly from './Icons/IconItaly';
import IconNetherlands from './Icons/IconNetherlands';
import IconSpain from './Icons/IconSpain';
import IconSwitzerland from './Icons/IconSwitzerland';
import IconUnitedKingdom from './Icons/IconUnitedKingdom';

const icons = [
  { icon: IconAbTest, name: 'IconAbTest' },
  { icon: IconAccount, name: 'IconAccount' },
  { icon: IconAlertCircle, name: 'IconAlertCircle' },
  { icon: IconAnalyse, name: 'IconAnalyse' },
  { icon: IconArrowFullDown, name: 'IconArrowFullDown' },
  { icon: IconArrowFullLeft, name: 'IconArrowFullLeft' },
  { icon: IconArrowFullRight, name: 'IconArrowFullRight' },
  { icon: IconArrowFullUp, name: 'IconArrowFullUp' },
  { icon: IconAward, name: 'IconAward' },
  { icon: IconBankTransfert, name: 'IconBankTransfert' },
  { icon: IconBarredEye, name: 'IconBarredEye' },
  { icon: IconBasket, name: 'IconBasket' },
  { icon: IconBathTub, name: 'IconBathTub' },
  { icon: IconBigDrill, name: 'IconBigDrill' },
  { icon: IconBox, name: 'IconBox' },
  { icon: IconBoxes, name: 'IconBoxes' },
  { icon: IconBrush, name: 'IconBrush' },
  { icon: IconBurgerMenu, name: 'IconBurgerMenu' },
  { icon: IconCalendar, name: 'IconCalendar' },
  { icon: IconCamera, name: 'IconCamera' },
  { icon: IconCheck, name: 'IconCheck' },
  { icon: IconChevronDown, name: 'IconChevronDown' },
  { icon: IconChevronLeft, name: 'IconChevronLeft' },
  { icon: IconChevronRight, name: 'IconChevronRight' },
  { icon: IconChevronUp, name: 'IconChevronUp' },
  { icon: IconCircleMinus, name: 'IconCircleMinus' },
  { icon: IconCirclePlus, name: 'IconCirclePlus' },
  { icon: IconClock, name: 'IconClock' },
  { icon: IconClose, name: 'IconClose' },
  { icon: IconCloseCircle, name: 'IconCloseCircle' },
  { icon: IconContent, name: 'IconContent' },
  { icon: IconCreditCard, name: 'IconCreditCard' },
  { icon: IconDeliveryForm, name: 'IconDeliveryForm' },
  { icon: IconDocument, name: 'IconDocument' },
  { icon: IconDotMenu, name: 'IconDotMenu' },
  { icon: IconDownload, name: 'IconDownload' },
  { icon: IconDragNdrop, name: 'IconDragNdrop' },
  { icon: IconDrill, name: 'IconDrill' },
  { icon: IconEdit, name: 'IconEdit' },
  { icon: IconExtra, name: 'IconExtra' },
  { icon: IconEye, name: 'IconEye' },
  { icon: IconFacebook, name: 'IconFacebook' },
  { icon: IconFaucetWater, name: 'IconFaucetWater' },
  { icon: IconFeatureFlag, name: 'IconFeatureFlag' },
  { icon: IconFilter, name: 'IconFilter' },
  { icon: IconFlash, name: 'IconFlash' },
  { icon: IconFreeDelivery, name: 'IconFreeDelivery' },
  { icon: IconHeart, name: 'IconHeart' },
  { icon: IconHeartFull, name: 'IconHeartFull' },
  { icon: IconHelpCircle, name: 'IconHelpCircle' },
  { icon: IconHistoricalSearch, name: 'IconHistoricalSearch' },
  { icon: IconHood, name: 'IconHood' },
  { icon: IconInformationCircle, name: 'IconInformationCircle' },
  { icon: IconInstagram, name: 'IconInstagram' },
  { icon: IconLabel, name: 'IconLabel' },
  { icon: IconLamp, name: 'IconLamp' },
  { icon: IconLaunch, name: 'IconLaunch' },
  { icon: IconLink, name: 'IconLink' },
  { icon: IconLock, name: 'IconLock' },
  { icon: IconLogOut, name: 'IconLogOut' },
  { icon: IconMail, name: 'IconMail' },
  { icon: IconMapPin, name: 'IconMapPin' },
  { icon: IconMarketing, name: 'IconMarketing' },
  { icon: IconMessage, name: 'IconMessage' },
  { icon: IconMessageCircle, name: 'IconMessageCircle' },
  { icon: IconPhone, name: 'IconPhone' },
  { icon: IconPiggyBank, name: 'IconPiggyBank' },
  { icon: IconPinterest, name: 'IconPinterest' },
  { icon: IconPlay, name: 'IconPlay' },
  { icon: IconPrinter, name: 'IconPrinter' },
  { icon: IconProduct, name: 'IconProduct' },
  { icon: IconPromo, name: 'IconPromo' },
  { icon: IconScrew, name: 'IconScrew' },
  { icon: IconSearch, name: 'IconSearch' },
  { icon: IconStarEmpty, name: 'IconStarEmpty' },
  { icon: IconStarFull, name: 'IconStarFull' },
  { icon: IconStop, name: 'IconStop' },
  { icon: IconTagPromo, name: 'IconTagPromo' },
  { icon: IconThumbDown, name: 'IconThumbDown' },
  { icon: IconTrash, name: 'IconTrash' },
  { icon: IconTrending, name: 'IconTrending' },
  { icon: IconTrucks, name: 'IconTrucks' },
  { icon: IconTwitter, name: 'IconTwitter' },
  { icon: IconVote, name: 'IconVote' },
  { icon: IconWall, name: 'IconWall' },
  { icon: IconWateringCan, name: 'IconWateringCan' },
  { icon: IconWheel, name: 'IconWheel' },
  { icon: IconYoutube, name: 'IconYoutube' },
  { icon: IconBelgium, name: 'IconBelgium' },
  { icon: IconFrance, name: 'IconFrance' },
  { icon: IconGermany, name: 'IconGermany' },
  { icon: IconItaly, name: 'IconItaly' },
  { icon: IconNetherlands, name: 'IconNetherlands' },
  { icon: IconSpain, name: 'IconSpain' },
  { icon: IconSwitzerland, name: 'IconSwitzerland' },
  { icon: IconUnitedKingdom, name: 'IconUnitedKingdom' },
];

const style = {
  display: 'flex',
  flexFlow: 'column wrap',
  justifyContent: 'center',
  alignItems: 'center',
  width: 140,
  height: 80,
  padding: '.5rem',
  margin: '0 .5rem .5rem 0',
  background: '#e8e8e8',
  border: '1px solid #d4d4d4',
  borderRadius: 3,
};

const span = {
  marginTop: 5,
  fontSize: 12,
  textAlign: 'center',
};

<div
  style={{
    display: 'flex',
    flexFlow: 'row wrap',
    justifyContent: 'flex-start',
  }}
>
  {icons.map(({ icon, name }) => (
    <div key={name} style={style}>
      <Icon component={icon} />
      <span style={span}>{name}</span>
    </div>
  ))}
</div>;
```
