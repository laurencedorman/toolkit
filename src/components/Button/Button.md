Default Button:

```js
import Button from '../Button';
import Layout from '../Layout';

<Layout styleGuidist direction="row">
  <Button>default Button</Button>
  <Button theme="primary">primary Button</Button>
  <Button theme="secondary">secondary Button</Button>
  <Button theme="ghost">ghost Button</Button>
  <Button theme="reset">reset style Button</Button>
  <Button theme="link">Link Button</Button>
  <Button disabled>disabled Button</Button>
</Layout>;
```

Button Size:

```js
import Button from '../Button';
import Layout from '../Layout';

<Layout styleGuidist direction="row">
  <Button size="big">big button</Button>
  <Button>default button</Button>
  <Button size="small">small button</Button>
  <Button size="xsmall">xsmall button</Button>
</Layout>;
```

Icon Button:

```js
import Button from '../Button';
import IconEdit from '../Icon/Icons/IconEdit';
import Layout from '../Layout';

<Layout styleGuidist direction="row">
  <Button size="big" iconComponent={IconEdit}>
    Button
  </Button>
  <Button iconComponent={IconEdit}>Button</Button>
  <Button size="small" iconComponent={IconEdit}>
    Button
  </Button>
  <Button size="xsmall" iconComponent={IconEdit}>
    Button
  </Button>
</Layout>;
```

Icon Reverse Button:

```js
import Button from '../Button';
import IconEdit from '../Icon/Icons/IconEdit';
import Layout from '../Layout';

<Layout styleGuidist direction="row">
  <Button theme="primary" size="big" iconComponent={IconEdit} reverse>
    Button
  </Button>
  <Button theme="primary" iconComponent={IconEdit} reverse>
    Button
  </Button>
  <Button theme="primary" size="small" iconComponent={IconEdit} reverse>
    Button
  </Button>
  <Button theme="primary" size="xsmall" iconComponent={IconEdit} reverse>
    Button
  </Button>
</Layout>;
```

Icon only Button:

```js
import Button from '../Button';
import IconEdit from '../Icon/Icons/IconEdit';
import Layout from '../Layout';

<Layout styleGuidist direction="row">
  <Button size="big" iconComponent={IconEdit} />
  <Button iconComponent={IconEdit} />
  <Button iconComponent={IconEdit} theme="reset" fill="#424242" />
  <Button size="small" iconComponent={IconEdit} />
  <Button size="xsmall" iconComponent={IconEdit} />
</Layout>;
```
