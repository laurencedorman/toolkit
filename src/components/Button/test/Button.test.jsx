/**
 * Copyright (c) Colibri SAS - ManoMano
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

import React from 'react';
import { shallow } from 'enzyme';
import Button from '../Button';
import IconEdit from '../../Icon/Icons/IconEdit';
import Icon from '../../Icon';

describe('Button', () => {
  it('should match snapshot', () => {
    // Given
    const wrapper = shallow(<Button />);
    // Then
    expect(wrapper).toMatchSnapshot();
  });

  it('should render a icon if props is true', () => {
    // Given
    const wrapper = shallow(<Button iconComponent={IconEdit} />);
    // Then
    expect(wrapper.find(Icon)).toHaveLength(1);
  });

  it('should render a title', () => {
    // Given
    const wrapper = shallow(<Button title="titleTest" />);
    // Then
    expect(wrapper.text()).toEqual('titleTest');
  });

  it('should render children', () => {
    // Given
    const wrapper = shallow(<Button>children</Button>);
    // Then
    expect(wrapper.text()).toEqual('children');
  });
});
