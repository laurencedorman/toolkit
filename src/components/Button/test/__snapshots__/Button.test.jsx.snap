// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match snapshot 1`] = `
<button
  className="toolkit button default default onlyIcon"
  data-qa=""
  disabled={false}
  style={Object {}}
  type="button"
/>
`;
